packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -129,14 +129,9 @@ export async function fetchHelper(
             duration: performance.now() - start,
         };
     }, (error) => {
-        // While we do not know for sure whether computer is offline, this error is not actionable and
-        // is pretty good indicator we are offline. Treating it as offline scenario will make it
-        // easier to see other errors in telemetry.
```

Comment:
I'm taking a quick look at Loop telemetry to see where offlineError v. fetchFailure show up just to get a picture of how this change will feel once it rolls out.

Reply:
Looks fine to me, this error will fit right in with the other `fetchFailure`s (in terms of meaning/interpretation).